hapi/src/utils/lifebankcode.js

```diff
@@ -29,7 +29,7 @@ const addDonor = (account, password) => {
 const addLifebank = (
   account,
   password,
-  { name, has_immunity_test, geolocation, ...profile }
+  { name, hasImmunityTest, geolocation, ...profile }
```

Comment:
@JustinCast the FE will send `has_immunity_test` https://github.com/eoscostarica/lifebank/blob/9c84bb7113a9ec226113f103b038ba1285a4cb02/webapp/src/routes/Signup/SignupLifeBank.js#L184 so if you are using a destructuring with `hasImmunityTest ` always will be undefined if you are trying to fix a warning related to snake_case you should use `has_immunity_test : hasImmunityTest`